Automattic/jetpack
# Search: automatically select product result format for sites with WooCommerce active

#### Changes proposed in this Pull Request:
For users setting up Jetpack Search, the default result format is 'expanded'.

This PR changes the default for sites with Woo installed and active. Those sites will use the new 'product' result format instead.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Apply this patch to your Jetpack site before purchasing Jetpack Search.
* Install the WooCommerce plugin on your site.
* Purchase the Jetpack Search subscription for your site.
* Perform a search and ensure that you are shown the 'product' result format.
* Check that Jetpack Search setup in the Customizer shows the 'product' result format as the current selection.

<img width="286" alt="Screen Shot 2021-03-09 at 15 25 11" src="https://user-images.githubusercontent.com/17325/110409517-4993eb00-80ec-11eb-89b7-1a0d9573c027.png">

Repeat the above steps on a new site without installing WooCommerce. The result format should be 'expanded' (the default). 

#### Proposed changelog entry for your changes:
* Search: during setup, automatically use the product result format for sites with WooCommerce active.
